package.json

```diff
@@ -42,7 +42,7 @@
     "micro": "^9.0.2",
     "micro-cors": "^0.0.4",
     "mime": "^2.1.0",
-    "open": "^0.0.5",
+    "opener": "^1.4.3",
```

Comment:
This is okay but should be a different pr

Reply:
Ops, sorry

Reply:
No problem at all 👌